plugins/woocommerce-admin/client/products/sections/pricing-section.tsx

```diff
@@ -91,6 +155,12 @@ export const PricingSection: React.FC = () => {
 	);
 	const salePriceProps = getInputProps( 'sale_price', currencyInputProps );
 
+	const dateTimePickerProps = {
+		className: 'woocommerce-product__date-time-picker',
```

Comment:
Just a general comment/question, I notice sometimes we've been using `woocommerce-product__` and `woocommerce-product-form__`.  

I don't have a strong opinion on which to use, but we might want to decide collectively on which to use going forward.  No need to hold up this PR though 😄

Reply:
💯 `woocommerce-product-form__` feels more specific, but I too don't feel strongly. I'll keep this as is for this PR and we can always have a followup PR to address consistency across the board.